docs/components/board/beaglebone.md

```diff
@@ -18,7 +18,9 @@ Follow the [setup guide](/installation/prepare/beaglebone-setup/) to prepare you
 
 {{% /alert %}}
 
-Configure a `beaglebone` board to integrate [BeagleBoard's BeagleBone AI 64](https://www.beagleboard.org/boards/beaglebone-ai-64) into your machine:
+Configure a `beaglebone` board to integrate [BeagleBoard's BeagleBone AI 64](https://www.beagleboard.org/boards/beaglebone-ai-64) into your machine.
+Make sure to first physically assemble the board and turn it on.
```

Comment:
For boards, I'm wondering if we should just say
```suggestion
Connect the board to power.
```
or nothing at all.
I'm not sure but I imagine most boards don't require much assembly--I know if you have a brand-new RPi you stick the heat sinks onto it but that's it, so maybe "assemble" doesn't make sense. Meanwhile, I don't want to get into installation instructions because that'd confuse the whole part vs board thing.

Often, as you configure digital interrupts, you'll move wires and double-check wired connections, and you don't want to have power to the board as you might short something. So I wonder if we should actually just leave all the board pages unchanged, because you wouldn't even want to connect it to power until after you configured it.

Reply:
You don't physically assemble a board I think ever? They come assembled? How about this?
```suggestion
To test your board as you configure it, power it on.
If you plan to connect hardware to the board's pins, connect the hardware while it's powered off.
```